templates/python/README.mustache

```diff
@@ -3,58 +3,25 @@
 {{{.}}}
 {{/appDescriptionWithNewLines}}
 
-This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
-
-- API version: {{appVersion}}
-- Package version: {{packageVersion}}
-{{^hideGenerationTimestamp}}
-- Build date: {{generatedDate}}
-{{/hideGenerationTimestamp}}
-- Generator version: {{generatorVersion}}
-- Build package: {{generatorClass}}
 {{#infoUrl}}
 For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
 {{/infoUrl}}
 
-## Requirements.
+This package is part of the STACKIT Python SDK. For additional information please visit the [github repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```

Comment:
```suggestion
This package is part of the STACKIT Python SDK. For additional information, please visit the [GitHub repository](https://{{gitHost}}/{{{gitUserId}}}/{{{gitRepoId}}}) of the SDK.
```

Reply:
We could also just write a static link to the repo, as this won't change

Reply:
We could. But I'm not sure what we would gain from this change. It is currently producing the link, and the generator is more general, so why change it to a static link?